assignments/this.js

```diff
@@ -1,26 +1,88 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Implicit Binding
+* 2. Explicit Binding
+* 3. New Binding
+* 4. Window Binding
 *
 * write out a code example of each explanation above
+Implicit Binding
 */
 
+
+
+
 // Principle 1
 
 // code example for Window Binding
+function getMessage(sentence) {
```

Comment:
This is not an example of Windows/Global binding of the `this` keyword because you don't use `this` at all 😞